dev.slimevr.SlimeVR.yml

```diff
@@ -0,0 +1,90 @@
+app-id: dev.slimevr.SlimeVR
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '42'
```

Comment:
current runtime is 43.

42 will EOL is April.

Reply:
I need to build a old version of WebKitGTK2 then

Reply:
It's kind of hard to do this. Do you have any guides on how I could do it? It has a lot of dependencies :sob:

Reply:
Will change it when the new tauri version gets released